[Beautification] Re-themed ENTIRE Project

MuhsinunCool opened the pull request:
Title says it all. I heard some people complaining about how they couldn't use the native BAT Web Interface because it was so ugly, so I decided to make it really pretty. I have everything setup to be based off of NamelessMC so everything matches up with NamelessMC's style. Enjoy! <3 :O :D

alphartdev commented:
Hi!

First of all thanks for that great job, it's really nice to see people participating to the project (even more as I barely support the webinterface).

I've made a list of a few points I'd like to discuss in your pull request:

- concerning the NamelessMcAddon, I don't know anything about Nameless MC, but it seems like a great software so that's nice
- the readme file has been deleted, it needs to be back up
- I don't really understand what you've done with Useful PHP.php file (even without mentioning that spaces in a filename is not  a great idea). If you want to add a note to other programmers, and not use it as a php file to be executed, change the extension to something else (DeveloperGuide.MD for example) or append it at the end of the readme.
- You removed the port in the mysql configuration, but some user don't use 3306, so we need to put it back as well.
- I think you removed the changepassword.php file, did you add it back somewhere else?
- Sometimes you chose to use "User Who Banned" and "User Who Revoked" instead of "staff" and "unban staff". I think it would be better to have the same thing everywhere and as in the ingame plugin it's staff and unban staff, to keep them and therefore remove the "User Who Banned" and "User Who Revoked".
- you kept the Composer file (vendor, autoload and so on), as those are supposed to be generated by the developer I'd prefer to keep them ouf ot the repo, because I don't really like build files in the repo
- you changed a few sentences or words, English being not my mother tongue I trust and thank you with that
- you added TONS of new CSS, I don't really see the point of that as you basically changed the colours and the navbar and some other minors changes but nothing requesting a lot of new CSS in my opinion. But if you really think it's needed, you did the developement work, feel free to keep them

Finally, basically all the projects has been changed as the code has been auto indented, so I might have missed a few changes, but I think that's it :)

Thanks again for you work, I'd be glad to pull these and upload it to spigot once the issues are resolved.

Friendly,
Alphart

P.S: Do you have this version in production and already thoroughly tested it or do I need to beta test it before uploading it to the whole spigot?

alphartdev commented:
Hi,
I don't know what you're trying to achivie with those moves but you posted your login informations in clear there: https://github.com/alphartdev/BAT-WebInterface/blob/64b130e04460d6c96f8c0dba2d986912896ccea3/web/application/config/config.inc.php?pr=/alphartdev/BAT-WebInterface/pull/17 (first commit).

So for the your setup security, I URGE you to CHANGE them.

MuhsinunCool commented:
Oh, sorry, I did those before actually reading your msgs. And the login infos are once again gone. Whew! Thanks for the heads up! That could've cost me my server!

Replies To Points:
- Check out NamelessMC, it's pretty dope lol
- Sorry about the README file, I'm assuming it just got lost as I worked on things haha
- The Useful PHP file was for developers, yes. It was there for me to look at if I were to ever come back to this project (which i actually AM planning to) so I could be caught back up to speed on some useful PHP (hence the name) & yes, I'll put them in an MD file. I just wanted the PHP formatting to show up in my coding software so I decided to just have a temporary PHP file there lol.
- I removed the port? Really? Don't remember doing that... interesting... I'll go check it out.
- Don't remember removing the changepassword.php file either... interesting... Btw, keep in mind that I didn't use a fork of your GitHub repository because I couldn't figure out how to get the Composer to work unless I used the version of this interface that I downloaded from spigotmc.org, so I just worked on THAT version... Hehe...
- I changed the messages to those because I didn't think most people would understand who those messages were referring to.
- Already explained the reasoning for the composer files for above... derp
- I haven't revisited this project in a while now, (apart from that derpy "move files" commit haha) so I don't actually remember exactly, but I'm assuming that they're needed. Soon, as I'm planning on revisiting this project, the CSS will be cut down DRAMATICALLY as I'll be re-theming the ENTIRE project... AGAIN... Haha! This is to suit my needs for my Minecraft server's forums software, which is NamelessMC.

The code wasn't auto-indented, no sir, no. That was so confusing lmao. I was struggling to understand the code so I decided to go thru every single file that I needed to mess with, indent & format them, and THEN begin coding on them. Lmao I can't stand it when developers don't indent their code, like the project that I'm currently working on; a custom Google Material Design template for NamelessMC 2.0.

I'm glad you enjoyed it. Please reply with anything else you need to say, and I'll go over these issues ASAP, although I have school too :P

P.S. PLEASE remove the message where u linked to my database credentials... Derp... Lmao. Tysm! :D Also, if you wanna look at the screenshots, checkout the SpigotMC page I made for the plugin last night. You seemed to be inactive and hadn't replied to my GitHub pull request (as far as I knew at the time) so I decided to make a SpigotMC resource posting of my version of the web interface. If you want, I can take it down, but I don't see any issues with it being up as I credited you in any way that I could.

I tested things as I went, although I didn't go crazy testing however, so I urge you to double check everything for any possible mistakes that I may have made. :P :D <3 Also, yea, don't upload it to SpigotMC until all the issues are accounted for. Thanks!
